### DISCUSSION TITLE
responseMocker() build error

### DISCUSSION THREAD

**Discussion by thedefside** [selenide/selenide#2226] [Q&A] [answered]
I'm trying to use the SelenideProxy to mock a response for my application. I used the example from the Selenide blog, but when I have `getSelenideProxy().responseMocker()` I get a build error:

`java: cannot access com.browserup.bup.filters.RequestFilter
class file for com.browserup.bup.filters.RequestFilter not found`

I'm sure it's something simple that I need to change in my dependencies configuration, but I haven't been able to figure it out. Any help would be very much appreciated!

**Comment by BorisOsipov** [comment, marked as answer]
@thedefside 
>I'm sure it's something simple that I need to change in my dependencies configuration

Just replace your `selenide` dependency to `selenide-proxy`

**Reply by thedefside** [reply]
I have selenide-testng as my only Selenide dependency in my pom.xml. Should I replace that with selenide-proxy?

**Reply by BorisOsipov** [reply]
>Should I replace that with selenide-proxy?

No. In such a case, add selenide-proxy along with selenide-testng of the same version.

**Reply by thedefside** [reply]
Thank you very much! Have you used this feature much? I am currently using the selenium NetworkInterceptor. It works, but has some issues. I was hoping to replace it with the Selenide proxy.

Also. do you know if it is possible to specify the response code in the responseMocker()? I was hoping to test some error states by returning mocked error responses.

Thanks again for your help!

**Reply by BorisOsipov** [reply]
>Have you used this feature much?

Nope.

>do you know if it is possible to specify the response code in the responseMocker()?

currently not

**Reply by asolntsev** [reply]
@thedefside It's a good idea (about response code), I have registered a feature request: https://github.com/selenide/selenide/issues/2227
